client/main.js

```diff
@@ -40,6 +41,7 @@ let render = () => {
 				storeOptions={ storeOptions }
 				schema={ formSchema }
 				layout={ formLayout }
+				countriesData={ countriesData }
```

Comment:
Instead of having countriesData at all, why not add it to storeOptions?  I would hate to see us get into a habit of adding too many objects that we end up needing to "wire through" to their consumers this way.